bw2h5.py

```diff
@@ -116,61 +119,68 @@ def parse_args():
 batch_ends = batch_starts + args.batch_size
 batch_ends[-1] = len(intervals)
 
-# Write batches to hdf5 file
-_logger.info('Extracting data for each batch and writing to h5 file')
+# Get output hdf5 filename
 df = None
 filename = args.prefix + '.h5'
+
+# Write batches to hdf5 file
+_logger.info('Extracting data for each batch and writing to h5 file')
 with h5py.File(filename, 'w') as f:
     # Create a single dataset -- expand locations as we go along
     for i in range(batches_per_epoch):
+        batch_name = 'batch' + str(i)
         if i % 10 == 0:
-            _logger.info("batch: " + str(i) + " of " + str(batches_per_epoch))
+            _logger.info(batch_name + " of " + str(batches_per_epoch))
+            batch_data = {}
```

Comment:
is this supposed to be initialized inside the `if` block ?

Reply:
fixed

Reply:
I meant the `batch_data` dictionary, sorry. Shouldn't it be defined outside the `if i % 10 == 0:` block ?

Reply:
just saw that you fixed it in other commit. Thanks